test/bun.js/os.test.js

```diff
@@ -125,6 +125,23 @@ it("networkInterfaces", () => {
   }
 });
 
+it("machine", () => {
+  const possibleValues = [
+    "arm",
+    "aarch64",
+    "mips",
+    "mips64",
+    "ppc64",
+    "ppc64le",
+    "s390",
+    "s390x",
+    "i386",
+    "i686",
+    "x86_64",
+  ];
+  expect(possibleValues.includes(os.machine())).toBe(true);
```

Comment:
I don't think this test is correct - wonder what Node does in its tests for this.

```
❯ sysctl machdep.cpu.brand_string
machdep.cpu.brand_string: Apple M1 Max
```

Reply:
I checked the source, they were using `ustname`. Couldn't find `ustname` anywhere in `std.c.darwin` or `std.os.darwin` in the Zig source either.

Reply:
Ah found the commit that added `uv_os_uname` to libuv https://github.com/buckaroo-pm/libuv/commit/1c66e34abeb95d1462a156c52e5f9d7d3ea81358 Kind of hard to understand that code tho.  I guess we would have to reproduce the code for getting machine on UNIX for darwin?